lib/galaxy/managers/workflows.py

```diff
@@ -90,9 +90,14 @@ def check_security(self, trans, has_workflow, check_ownership=True, check_access
         if not check_ownership or check_accessible:
```

Comment:
Hmm, this break the subworkflow tests. Subworkflows call `get_owned_workflow` which calls `check_security` like this: `self.check_security(trans, workflow, check_ownership=True)`, which effectively used to skip all checks. This fails now when looking for `has_workflow.top_level_stored_workflow`